### ISSUE TITLE
Chat Messages

### ISSUE THREAD

**Issue by lucariojb** [graywolf336/CasinoSlots#76]
Hey,
I can't find the Message for a Slot, what you have to play with an Item
For Example you need an Diamond to play, the Message is "1 diamond removed from your Inventory."
Where can I change this message? I searched in the Configs but did not find it.
Sorry for my bad english... im not so good.. :(

**Comment by graywolf336** [comment]
Yeah, that one isn't configurable in v2. I'll make sure this is enabled in v3.
